integrationlink/create.go

```diff
@@ -28,10 +28,7 @@ func Create(app string, integrationType scalingo.SCMType, integrationURL string,
 		return errgo.Notef(err, "fail to get the integration")
 	}
 
-	repoLink, err := c.SCMRepoLinkShow(app)
-	if err != nil {
```

Comment:
I don't really like that you completely remove the error checking. Can't you just ignore the error in case it's a 404?